learning_journal/tests.py

```diff
@@ -1,54 +1,65 @@
-import pytest
```

Comment:
You appear to have replaced all your pytest tests with the scaffolding tests.  Oooops.